Doc/library/sys.rst

```diff
@@ -729,34 +729,41 @@ always available.
-   +---------------------+--------------------------------------------------+
-   | :const:`nan`        | hash value returned for a nan                    |
-   +---------------------+--------------------------------------------------+
-   | :const:`imag`       | multiplier used for the imaginary part of a      |
-   |                     | complex number                                   |
-   +---------------------+--------------------------------------------------+
-   | :const:`algorithm`  | name of the algorithm for hashing of str, bytes, |
-   |                     | and memoryview                                   |
-   +---------------------+--------------------------------------------------+
-   | :const:`hash_bits`  | internal output size of the hash algorithm       |
-   +---------------------+--------------------------------------------------+
-   | :const:`seed_bits`  | size of the seed key of the hash algorithm       |
-   +---------------------+--------------------------------------------------+
+   +------------------------+-----------------------------------------------+
+   | attribute              | explanation                                   |
+   +========================+===============================================+
+   | :const:`width`         | width in bits used for hash values            |
+   +------------------------+-----------------------------------------------+
+   | :const:`modulus`       | prime modulus P used for numeric hash scheme  |
+   +------------------------+-----------------------------------------------+
+   | :const:`inf`           | hash value returned for a positive infinity   |
+   +------------------------+-----------------------------------------------+
+   | :const:`nan`           | hash value returned for a nan                 |
+   +------------------------+-----------------------------------------------+
+   | :const:`imag`          | multiplier used for the imaginary part of a   |
+   |                        | complex number                                |
+   +------------------------+-----------------------------------------------+
+   | :const:`algorithm`     | name of the algorithm for hashing of str,     |
+   |                        | bytes, and memoryview                         |
+   +------------------------+-----------------------------------------------+
+   | :const:`hash_bits`     | internal output size of the hash algorithm    |
+   +------------------------+-----------------------------------------------+
+   | :const:`seed_bits`     | size of the seed key of the hash algorithm    |
+   +------------------------+-----------------------------------------------+
+   | :const:`use_hash_seed` | Whether to use :envvar:`PYTHONHASHSEED`       |
+   +------------------------+-----------------------------------------------+
+   | :const:`hash_seed`     | The value from :envvar:`PYTHONHASHSEED`       |
```

Comment:
At the C level, we need two fields, since it's not possible to distinguish "value set" to "value unset". But at the Python level, we can use None when the hash seed is unset.

Reply:
It seems like the seed is always in the range [0; 4294967295], _Py_ReadHashSeed:

```
        if (*endptr != '\0'
            || seed > 4294967295UL
            || (errno == ERANGE && seed == ULONG_MAX))
        {
            return -1;
        }
```

It may be worth to document the value range.